src/cartesian/CartesianGrid.tsx

```diff
@@ -37,6 +37,9 @@ interface CartesianGridProps extends InternalCartesianGridProps {
   verticalPoints?: number[];
   verticalFill?: string[];
   horizontalFill?: string[];
+  syncWithTicks?: boolean;
+  horizontalValues?: number[] | string[];
+  verticalValues?: number[] | string[];
```

Comment:
This is a great PR, it adds value and is concise. Lets still reflect a second on the API. Is this consistent with the rest of the API? Can it be done more concisely?

Questions: 

(1) Why would the horizontalValues be a string? 
(2) What do `horizontalPoints` do vs `horizontalValues`? Both seem quite generic in name.
(3) Do you see syncWithTicks to be extended in the near future? What are feature that we already know we want to add here? (are there any that you could think of)

Change request: 
- Please add comments above each new field in the form of
```
  /** Some helpful explanation */
  syncWithTicks?: boolean;
```

These comments would then automatically show up in storybook in the ArgsTable, which would help us document the API.

Reply:
1. YAxis type could be 'number' | 'category'. So, horizontal values could be strings, if we will have YAxis type as category.
2. HorizontalPoints are array of coordinates (in pixels), while ...values are axis values. I think it's hard sometimes to calc coordinates yourself.
3. Honestly, I have no idea about it.